routes/web.php

```diff
@@ -30,26 +33,72 @@
 Route::post('/contact', [ContactUsController::class, 'store'])
     ->name('contact.store');
 
-Route::get('/films/create', [FilmController::class, 'createForm'])
-    ->name('film.create.form');
 
-Route::post('/films/create', [FilmController::class, 'create'])
-    ->name('film.create');
+Route::group(['middleware' => ['auth']], function () {
+    Route::get('/films/create', [FilmController::class, 'createForm'])
+        ->name('film.create.form');
 
-Route::get('/films', [FilmController::class, 'list'])
-    ->name('film.list');
+    Route::post('/films/create', [FilmController::class, 'create'])
+        ->name('film.create');
 
-Route::get('/films/{id}', [FilmController::class, 'show'])
-    ->name('film.show');
+    Route::get('/films', [FilmController::class, 'list'])
+        ->name('film.list');
 
-Route::get('/films/{id}/edit', [FilmController::class, 'editForm'])
-    ->name('film.edit.form');
+    Route::get('/films/{id}', [FilmController::class, 'show'])
+        ->name('film.show');
 
-Route::post('/films/{id}/edit', [FilmController::class, 'edit'])
-    ->name('film.edit');
+    Route::get('/films/{id}/edit', [FilmController::class, 'editForm'])
+        ->name('film.edit.form');
 
-Route::post('/films/{id}/delete', [FilmController::class, 'delete'])
-    ->name('film.delete');
+    Route::post('/films/{id}/edit', [FilmController::class, 'edit'])
+        ->name('film.edit');
+
+    Route::post('/films/{id}/delete', [FilmController::class, 'delete'])
+        ->name('film.delete');
+
+    Route::get('/actors/create', [ActorController::class, 'createForm'])
+        ->name('actor.create.form');
+
+    Route::post('/actors/create', [ActorController::class, 'create'])
+        ->name('actor.create');
+
+    Route::get('/actors', [ActorController::class, 'list'])
+        ->name('actor.list');
+
+    Route::get('/actors/{id}', [ActorController::class, 'show'])
+        ->name('actor.show');
+
+    Route::get('/actors/{id}/edit', [ActorController::class, 'editForm'])
+        ->name('actor.edit.form');
+
+    Route::post('/actors/{id}/edit', [ActorController::class, 'edit'])
+        ->name('actor.edit');
+
+    Route::post('/actors/{id}/delete', [ActorController::class, 'delete'])
+        ->name('actor.delete');
+
+
+    Route::get('/zhanrs/create', [ZhanrController::class, 'createForm'])
+        ->name('zhanr.create.form');
```

Comment:
```suggestion
    Route::get('/genres/create', [ZhanrController::class, 'createForm'])
        ->name('zhanr.create.form');
```

остальные тоже переименовать надо